src/integrationTest/java/uk/gov/hmcts/darts/authentication/controller/impl/LoginOrRefreshIntTest.java

```diff
@@ -35,6 +36,8 @@ class LoginOrRefreshIntTest {
 
     @MockBean
     private CourthouseRepository courthouseRepository;
```

Comment:
is this repository need for this test? I can see there is a notification repo and audio repo as well.